src/test/scala/cromwell/engine/backend/BackendCallSpec.scala

```diff
@@ -21,7 +21,7 @@ class BackendCallSpec extends CromwellTestkitSpec with ScalaFutures {
   "BackendCall hash function" should {
     "not change very often - if it changes, make sure it is for a good reason" in {
       val actual = backendCall.hash.futureValue.overallHash
-      val expected = "f91d97758ef2a97e9a4af15ad61bf1e8"
+      val expected = "9eb5cdfbb16dcccb0ca413a5a101ca7c"
```

Comment:
Actually there was a bug in the hash generation see this line https://github.com/broadinstitute/cromwell/pull/470/files#diff-ba03f845a104d1ad29451c15da6ba07bL163

In the string

```
s"${d.getName} ${d.size} ${d.getType}"
```

Where `d` is a `Disk` (One of Google's objects).  However, `d.size` is not the disk size, it's actually [this size function](https://docs.oracle.com/javase/7/docs/api/java/util/AbstractMap.html#size()).  The real size function is actually `getSizeGb()`.

So to fix this, I just stringify the `JesAttachedDisk` object which will output a string that one would use for the `disk` runtime attribute.